codespell_lib/data/dictionary.txt

```diff
@@ -51417,9 +51517,31 @@ terminats->terminates
 termindate->terminate
 termine->determine
 termined->terminated
+terminial->terminal
+terminially->terminally
+terminials->terminals
+terminiate->terminate
+terminiated->terminated
+terminiates->terminates
+terminiating->terminating
+terminiation->termination
+terminiations->terminations
 terminiator->terminator
-terminte->terminate
+terminiators->terminators
+terminl->terminal
```

Comment:
Perhaps `termini` (plural of `terminus`)?

Reply:
I couldn't find this typo.